nixos/modules/services/monitoring/grafana-alloy.nix

```diff
@@ -0,0 +1,80 @@
+{ lib, pkgs, config, ... }:
```

Comment:
@flokli Thanks for change, I was thinking if `grafana-alloy.nix` can go under `nixos/modules/services/logging` instead of `nixos/modules/services/monitoring` as previously [promtail.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/logging/promtail.nix) was under same directory and alloy will replace promtail later on.

Reply:
It can also act as a replacement for grafana-agent, prometheus, otlp-collector etc.

It can collect metrics, logs, traces and profiles, it's definitely not just about collecting logs.

Reply:
@vunso Promtail likely won't be replaced by Alloy; Alloy uses Promtail under the hood for all its `loki.source`, `loki.process` and `loki.relabel` blocks.

Reply:
I meant to say, you can use alloy as a replacement for promtail on your machine (and other collectors for other things). It's not obsoleting Promtail upstream.